### PR TITLE
Makes query bar stick to top on scroll

### PR DESCRIPTION
**Pull request by Zachary-Dernar**
Adds react-stickynode component

### CODE REVIEW

File: client/knowledge_base_search/src/client/knowledge_base_search/node_modules/.bin/loose-envify [hunk 1]
```diff
@@ -0,0 +1 @@
+../loose-envify/cli.js
```
Comment (maniax89) [inline client/knowledge_base_search/src/client/knowledge_base_search/node_modules/.bin/loose-envify]: what is this?

File: client/knowledge_base_search/src/App.js [hunk 2]
```diff
@@ -130,12 +131,14 @@ class App extends Component {
           version='GA'
           description='This starter kit demonstrates how Watson Discovery&#39;s Passage Search quickly finds the most relevant information in your documents to answer your natural language questions. Try out the preset questions or enter a custom question and compare the answers returned by a Standard (non-Passage) Search vs. a Passage Search on the Stack Exchange Travel data set.'
         />
-        <SearchContainer
-          onSubmit={this.handleSearch}
-          hasResults={this.state.results_fetched}
-          search_input={this.state.search_input}
-          isFetching={this.state.fetching}
-        />
+        <Sticky enabled={true} top=''>
```
Comment (maniax89) [inline client/knowledge_base_search/src/App.js]: if `top` is set to '', can we just remove it completely (says default is `0`)? same with `enabled` (says default is `true`)?

Comment (maniax89) [reply client/knowledge_base_search/src/App.js]: https://github.com/yahoo/react-stickynode#props

File: client/knowledge_base_search/src/client/knowledge_base_search/node_modules/.bin/loose-envify (hunk 1 above)
Comment (Zachary-Dernar) [reply client/knowledge_base_search/src/client/knowledge_base_search/node_modules/.bin/loose-envify]: I don't know. I think it was a result of installing the dependencies for react-stickynode

File: client/knowledge_base_search/src/App.js [hunk 3]
```diff
@@ -130,12 +131,14 @@ class App extends Component {
...
-          hasResults={this.state.results_fetched}
-          search_input={this.state.search_input}
-          isFetching={this.state.fetching}
-        />
+        <Sticky>
+          <SearchContainer
+            onSubmit={this.handleSearch}
+            hasResults={this.state.results_fetched}
+            search_input={this.state.search_input}
+            isFetching={this.state.fetching}
```
Comment (besh) [inline client/knowledge_base_search/src/App.js:139]: you could cache state here or deconstruct it above.

**Review by maniax89 — APPROVED** [review]

**Pull request merged**